CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - [[PR519]] (https://github.com/lanl/singularity-eos/pull/519) Split eos_spiner.hpp into eos_spiner_rho_sie.hpp and eos_spiner_rho_temp.hpp files.  
 
 ### Infrastructure (changes irrelevant to downstream codes)
+- [[PR528]](https://github.com/lanl/singularity-eos/pull/528) Remove unused variable coldGroup in SpinerEOSDependsRhoT
```

Comment:
I think since this is merged I can't suggest a change, but can you change this to reflect that you reoved `subGroup`?

Reply:
Never mind... you already had the MR in ;-)